k8s-e2e-external-storage.groovy

```diff
@@ -0,0 +1,127 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/ceph/ceph-csi'
+def ci_git_branch = 'ci/centos'
+def git_repo = 'https://github.com/ceph/ceph-csi'
+def ref = "master"
+def git_since = 'master'
+def skip_e2e = 0
+def doc_change = 0
+def k8s_release = 'latest'
+def namespace = 'k8s-e2e-storage-' + UUID.randomUUID().toString().split('-')[-1]
+
+def ssh(cmd) {
+	sh "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} '${cmd}'"
+}
+
+node('cico-workspace') {
+	stage('checkout ci repository') {
+		git url: "${ci_git_repo}",
+			branch: "${ci_git_branch}",
+			changelog: false
+	}
+
+	stage('skip ci/skip/e2e label') {
+		if (params.ghprbPullId == null) {
+			skip_e2e = 1
+			return
+		}
+		skip_e2e = sh(
```

Comment:
@nixpanic still need to update scripts to add GitHub token to avoid rate limit issue?

Reply:
Oh, yes, forgot about that!